wdl/imputation.wdl

```diff
@@ -558,12 +558,21 @@ task duplicates {
             awk -v batch=$batch '{OFS="\t"; print batch,$0}' $batch.samples_include.txt >> all_batches.samples_include.txt
             awk -v batch=$batch '{OFS="\t"; print batch,$0}' $batch.samples_exclude.txt >> all_batches.samples_exclude.txt
         done
+          #Rplot fix for batches that have empty exclusion list
+          awk -v batch=$batch 'BEGIN {FS="\t";OFS="\t"} { if ($1==batch) print "exists"; else print "not_exist"}' all_batches.samples_exclude.txt > file1.txt
+           VAR=$(grep -E "not_exist" file1.txt| wc -l); echo $VAR
+           if [[ $VAR -gt 0 ]]; then
+           echo "batch does not exist in excluded samples"
+           echo -e "$batch\tFG00000000\tlist1\tNA" >> all_batches.samples_exclude.txt
```

Comment:
Where does the empty exclusion list cause a problem? Possibly better to fix the issue there instead of adding fake ids?

Reply:
Dataset NFBC86 could not pass imputation pipeline because there are no samples to exclude and this results in error in step "plots". So we fixed this module to deal this issue and tested it for different single batches but we did not run it for multiple batches simulatenpuly, there possibly it can cause further issues as you wrote above. 
So we should fix this problem in task plots (in r script)?